common/filesystem.h

```diff
@@ -286,16 +283,16 @@ class Internal::FileRefBase {
   // handle in that case. This is to support rebinding operations.
   FileRefBase() = default;
 
-  // Computes the file status.
+  // Reads the file status.
   //
   // Analogous to the Unix-like `fstat` call.
   auto Stat() -> ErrorOr<FileStatus, FdError>;
 
   // Methods to seek the current file position, with various semantics for the
   // offset.
   auto Seek(int64_t delta) -> ErrorOr<int64_t, FdError>;
-  auto SeekTo(int64_t pos) -> ErrorOr<int64_t, FdError>;
-  auto SeekFromEnd(int64_t delta) -> ErrorOr<int64_t, FdError>;
+  auto SeekFromBeginning(int64_t pos) -> ErrorOr<int64_t, FdError>;
```

Comment:
nit:
```suggestion
  auto SeekFromBeginning(int64_t delta_from_beginning) -> ErrorOr<int64_t, FdError>;
```
Would more closely parallel the SeekFromEnd